### DISCUSSION TITLE
Getting forgot password links to point to /admin?

### DISCUSSION THREAD

**Discussion by brettruchan** [decaporg/decap-cms#7104] [Q&A]
Hello!  We're using DecapCMS and host our documentation on Netlify.  In Netlify, we use the Identity feature to add users to Decap.  However, when a user (or an administrator) resets a user's password, the link goes to the developers.ourdomain.com, where it should be going to developers.ourdomain.com/admin

What we have:
https://developers.ourdomain.com/#recovery_token=abcdefg...

What we need:
https://developers.ourdomain.com/admin#recovery_token=abcdefg...

My apologies if this is a question for Netlify, but if is a Decap setting, where would I go to fix this?

**Comment by twilightscapes** [comment]
I too, would like to know how to handle this. It's not just the recovery email, but all of the emails, from account creation to authorization.

**Comment by martinjagodic** [comment]
Use custom email templates in Netlify Identity settings: https://docs.netlify.com/security/secure-access-to-sites/identity/identity-generated-emails/
